src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  &--1 > .stars__star:nth-child(-n + 1),
+  &--2 > .stars__star:nth-child(-n + 2),
+  &--3 > .stars__star:nth-child(-n + 3),
+  &--4 > .stars__star:nth-child(-n + 4),
+  &--5 > .stars__star:nth-child(-n + 5) {
+    background-image: url(/src/images/star-active.svg);
```

Comment:
As with the regular star image, use a relative path for the active star image URL to maintain consistency and reliability across different environments. For example, `background-image: url(../../images/star-active.svg);`.